style.scss

```diff
@@ -140,13 +140,14 @@ img {
   z-index: 9999;
   color: var(--gray1);
   background-color: var(--gray5);
-  box-shadow: 0 4px 6px #80808033; // gray3 but with 20% alpha
+  box-shadow: 4px 4px 6px #80808033; // gray3 but with 20% alpha
```

Comment:
4px X offset? IMO it's better with only Y

Reply:
idk

Reply:
Okay wait